configs/or1k-zephyr-elf.config

```diff
@@ -0,0 +1,6 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+CT_ARCH_OR1K=y
```

Comment:
Does not look like something in ct-ng, at least the version we use. Looked at the upstream ct-ng and later version would have this, but then with `CT_ARCH_OPENRISC`